### ISSUE TITLE
Release Version for Windows

### ISSUE THREAD

**Issue by Makurisan** [zsa/kontroll#17]
Is there a release version for Windows?

**Comment by corbob** [comment]
Only 1.0.4 lacked a Windows release. You can get 1.0.3 here: https://github.com/zsa/kontroll/releases/tag/1.0.3

**Comment by fdidron** [comment]
Yes the Windows version didn't get a new release since 1.0.4 applied to macOS and Linux
